src/print/index.ts

```diff
@@ -114,49 +114,43 @@ export function definePrinter(options: Options): void {
   /** Prints embedded GlimmerExpressions/GlimmerTemplates. */
   printer.embed = (
     path: AstPath<BaseNode | undefined>,
-    _print: (path: AstPath<BaseNode | undefined>) => doc.builders.Doc,
-    textToDoc: (text: string, options: Options) => doc.builders.Doc,
     embedOptions: Options
```

Comment:
signature of `embed` changed, now expects an async function as the return value

https://prettier.io/docs/en/plugins.html#optional-embed